README.md

```diff
@@ -1,102 +1,38 @@
 # TypeSpec
 
-[Try TypeSpec Online](https://aka.ms/trytypespec)
+[Official Docs](https://typespec.io/) | [Try TypeSpec Online](https://aka.ms/trytypespec) | [Getting Started](https://typespec.io/docs) | [Language Overview](https://typespec.io/docs/language-basics/overview)
 
-TypeSpec is a language for describing cloud service APIs and generating other API
-description languages, client and service code, documentation, and other assets.
-TypeSpec provides highly extensible core language primitives that can describe API
-shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
+TypeSpec is a language for defining cloud service APIs and shapes. TypeSpec is a highly extensible language with primitives that can describe API shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
```

Comment:
```suggestion
TypeSpec is a language for defining cloud service APIs and shapes. TypeSpec is a highly extensible language with primitives that can describe API shapes common among REST, OpenAPI, gRPC, and other protocols.
```

I've wanted to drop this reference for a while, seems like now is a good time to do it (just because we don't have stdlib support, so it's weird to mention)